index.js

```diff
@@ -194,7 +194,16 @@ module.exports = function inspect_(obj, options, depth, seen) {
     }
     if (typeof obj === 'object' && customInspect) {
         if (inspectSymbol && typeof obj[inspectSymbol] === 'function') {
-            return obj[inspectSymbol]();
+            return obj[inspectSymbol](maxDepth - depth, {
+                breakLength: 60,
```

Comment:
hardcoding these defaults seems problematic - i do get that node probably passes them explicitly. is there a way we could look up the values so that if node changes them, ours change too?

Reply:
🤔 They could be retrieved by doing a dummy inspect call.

Reply:
`util.inspect.defaultOptions` might be helpful here